src/structured-data/structured-data-tags.js

```diff
@@ -141,8 +141,8 @@ function findStoryElementField(card, type, field, defaultValue) {
 
 function generateLiveBlogPostingData (structuredData = {}, story = {}, publisherConfig = {}){
   return {
-    "coverageEndTime": stripMillisecondsFromTime(new Date(story['last-published-at'])),
-    "coverageStartTime": stripMillisecondsFromTime(new Date(story['created-at'])),
+    "coverageEndTime": stripMillisecondsFromTime(new Date(story['updated-at'])),
```

Comment:
dont think this change is required.

Reply:
created and update time provide same timestamp. it is requested that it shouldn't be same